build.gradle.kts

```diff
@@ -29,6 +29,7 @@ allprojects {
     repositories {
         mavenCentral()
         maven("https://oss.sonatype.org/content/repositories/snapshots/")
+        maven("https://maven.enginehub.org/repo")
```

Comment:
Limit the repository to EngineHub dependencies only.

Reply:
Do you mind resolving this anytime soon? I'd like to merge this asap @TheMeinerLP

Reply:
Now :D